src/common/apiBase.ts

```diff
@@ -152,7 +153,9 @@ export class ApiBase {
         request.query(requestOptions.query);
 
         // set header parameters
+        const packInfo = packageInformation as any;
```

Comment:
nit: Prefer to see this typed if possible using  rather than using `any`. I believe this is possible by `declare`ing the type being imported.